docs/architecture/zero-token-nodes.rst

```diff
@@ -0,0 +1,15 @@
+=========================
+Zero-token Nodes
+=========================
+
+By default, all nodes in a cluster own a token range and are used to replicate
```

Comment:
The phrase "a token range" is slightly inaccurate in the case of tablets, because with tablets there is is a different subset of tokens owned by this node for each different table.

By the way, even with the old vnodes technique, what each node held was a _set_ of token ranges - not one contiguous token range.

But these are just small inaccuracies, the main point is that the zero-token nodes have no tokens in no tables, i.e., no data.